src/components/openapi/SchemaWithExample.tsx

```diff
@@ -8,6 +8,7 @@ import CodeBlock from "@theme/CodeBlock";
 
 interface Props {
   schema: OpenAPIV3.SchemaObject;
+  format?: "json" | "yaml";
```

Comment:
Maybe use a literal type here ? Just in case we want to add up another format or so.

Reply:
The type should be in the other component and reused here though

Reply:
Fixed ✅